cmd/eval_test.go

```diff
@@ -305,24 +305,32 @@ func TestEvalWithInvalidInputFile(t *testing.T) {
 	}
 }
 
-func testEvalWithSchemaFile(t *testing.T, input string, query string, schema string) error {
+func testEvalWithSchemaFile(t *testing.T, input string, query string, schema string, policy string, expTypeErr bool) error {
 	files := map[string]string{
 		"input.json":  input,
 		"schema.json": schema,
 	}
 
+	policyFilePresent := len(policy) > 0
```

Comment:
[nit/style] I'd use this to give an indication that we're dealing with a string here, not some slice. But it's not important:
```suggestion
	policyFilePresent := policy != ""
```